### PR TITLE
GO widget on gene page

### PR DESCRIPTION
**Pull request by sibyl229**
Hi, this PR completes the Gene Ontology widget on Gene page. Could @a8wright or @mgrbyte help review this? Thanks!

### CODE REVIEW

File: src/datomic_rest_api/rest/fields/gene.clj [hunk 1]
```diff
@@ -1320,11 +1320,48 @@
    :go-term.type/cellular-component "Cellular_component"
    :go-term.type/biological-process "Biological_process"})
 
+(defn- go-anno-extensions [anno]
+  (->>
+   (concat
+    (for [{rel :go-annotation.molecule-relation/text
```
Comment (mgrbyte) [inline src/datomic_rest_api/rest/fields/gene.clj]: There's a repeated pattern here that could be extracted to a small function:

```clojure
(defn- rel-pack-anno 
   [anno rel target anno-key]
   (for [{rel target} (anno-key anno)]
     [rel (pack-obj target)]))
```
```clojure
;; example usage
(->> (concat
          (rel-pack :go-annotation.molecule-relation/text
                         :go-annotation.molecule-relation/molecule
                         :go-annotation/molecule-relation)
           ...)

File: src/datomic_rest_api/rest/fields/gene.clj [hunk 2]
```diff
@@ -1451,10 +1477,10 @@
                 [?tdiv :db/ident ?div]]
        db (:db/id gene))
     (map
-     (fn [[div term code anno]]
+     (fn [[div term anno]]
        {:division div
-        :term     (entity db term)
-        :anno     (entity db anno)}))
+        :term (entity db term)
```
Comment (mgrbyte) [inline src/datomic_rest_api/rest/fields/gene.clj]: I'd probably stick to using the `d` namespace for the datomic api here.
e.g `(d/entity db term)`
(minor nit-pick)

**Review by mgrbyte — CHANGES_REQUESTED** [review]
Generally looks good to me.

Changing to use d/entity, d/q etc not _required_ 
for merge.

**Comment by sibyl229** [comment]
Thanks @mgrbyte for the review. The two improvements you suggested has been implemented.

**Pull request merged**